lms/static/js/groups/collections/cohort.js

```diff
@@ -0,0 +1,16 @@
+var edx = edx || {};
+
+(function(Backbone, CohortModel) {
+    'use strict';
```

Comment:
@cahrens These should be addressed in commit: https://github.com/edx/edx-platform/commit/49c48b265561d86913b9ca45e55bc4927ff969a8